pkg/admin/facade/session.go

```diff
@@ -10,7 +10,7 @@ type SessionManager interface {
 	List(userID string) ([]session.ListableSession, error)
 	Get(id string) (session.ListableSession, error)
 	RevokeWithEvent(session session.SessionBase, isTermination bool, isAdminAPI bool) error
-	TerminateAllExcept(userID string, currentSession session.Session, isAdminAPI bool) error
+	TerminateAllExcept(userID string, currentSession session.ResovledSession, isAdminAPI bool) error
```

Comment:
typo, It should be ResolvedSession.

Reply:
Fixed with a fixup to [6b34b37](https://github.com/authgear/authgear-server/pull/4370/commits/6b34b375ad1a4c81db4607f154513ce7b8b829cc)